lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -30,8 +30,7 @@ namespace Org.Apache.REEF.Network.Group.Task.Impl
     /// Handles incoming messages sent to this Communication Group.
     /// Writable version
     /// </summary>
-    // TODO: Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295.
-    public sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
+    internal sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
```

Comment:
done and pushed

On Wed, Jul 15, 2015 at 1:31 PM, Julia Wang <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs
> <https://github.com/apache/incubator-reef/pull/289#discussion_r34725318>:
>
> > @@ -30,8 +30,7 @@ namespace Org.Apache.REEF.Network.Group.Task.Impl
> >      /// Handles incoming messages sent to this Communication Group.
> >      /// Writable version
> >      /// </summary>
> > -    // TODO: Need to remove Iwritable and use IstreamingCodec. Please see Jira REEF-295.
> > -    public sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
> > +    internal sealed class CommunicationGroupNetworkObserver : ICommunicationGroupNetworkObserver
>
> You can prefix ICommunicationGroupNetworkObserver.Register() and remove
> public from the Register() method in CommunicationGroupNetworkObserver.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/289/files#r34725318>.
>